source/_integrations/lektrico.markdown

```diff
@@ -24,6 +24,25 @@ The Lektrico Charging Station device will be added as a sensor in Home Assistant
 
 {% include integrations/config_flow.md %}
 
+## Binary sensors
+
+Binary sensors available in the library:
+
+### Chargers
+
+| Condition         | Description                |
+| :---------------- | :------------------------- |
+| state_e_activated | State machine E activated  |
+| overtemp          | Overtemperature            |
+| critical_temp     | Critical temperature       |
+| overcurrent       | Overcurrent                |
+| meter_fault       | Meter fault                |
+| undervoltage      | Undervoltage               |
+| overvoltage       | Overvoltage                |
+| rcd_error         | RCD error                  |
+| cp_diode_failure  | CP diode failure           |
```

Comment:
what do RCD and CD stand for? 
would it make sense to spell them out? for example:
<abbr title="current transformers">CT</abbr>

Reply:
RCD = residual current device, it's a standard term.
Description "CP diode failure" was changed to "Electric vehicle communication error".
Thank you.